ethstaker_deposit/utils/validation.py

```diff
@@ -207,13 +209,18 @@ def validate_deposit_amount(amount: str) -> int:
         decimal_ether = Decimal(amount)
         amount_gwei = decimal_ether * Decimal(ETH2GWEI)
 
+        params = kwargs.get('params', {})
+        chain = params.get('chain', 'mainnet')
+        chain_setting = get_chain_setting(chain)
```

Comment:
How is the validator_deposit_amount supposed to obtain a custom devnet settings passed with `--devnet_chain_setting` if chain value only contains a named network provided in `settings.py` and `get_chain_setting` only retreiving chain settings for known named networks?